atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -206,13 +236,18 @@ def _get_pullspecs(self, operator_csv):
         pullspec_set = set()
 
         if not operator_csv.has_related_images():
+            if skip_all and operator_csv.get_pullspecs():
+                raise RuntimeError("skip_all defined but relatedImages section doesn't exist")
```

Comment:
do we care here?

Reply:
yes we do, because those pullspecs will be used in metadata, so we still need that basic check that
related images are defined, unless there aren't any pullspecs at all